### DISCUSSION TITLE
🧩 FEATURE REQUEST: Custom feeds

### DISCUSSION THREAD

**Discussion by argousiere** [dailydotdev/daily#476] [Feature Requests]
## Feature Request

I would like to organize my reading into several feeds, where each feed shows articles based on a set of tags. This would be a complement to the main feed where all articles based on my favorite tags is shown.

### Description

This will allow me to create a feed on front-end development, job interviewing, react, for example. I will be able to switch between them if I feel like focusing my reading only on a specific topic today, for example.

For now, a set of tags per feed would be great, but in the future, other search options can be saved, such as sorting, sources, keywords, etc.

Actions:
- name and rename the feed
- add/delete tags
- modify search options
- delete the feed

### Solution

I'm thinking of a section in the sidebar where all my custom feeds are listed. Each custom feed will have its own page, and I can edit things such as name and search options, and save these changes.

**Comment by idoshamun** [comment]
Hi @argousiere,

Thanks for the feature request. 😁 
We have it on our roadmap! Can't promise anything in terms of timeline though.
